### PR TITLE
Proc splunk 1

### PR DESCRIPTION
**Pull request by stevsmit**
https://issues.redhat.com/browse/PROJQUAY-5058

Preview: https://stevsmit.github.io/quay-docs/splunk

### CODE REVIEW

File: modules/proc_splunk-config.adoc [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+LOGS_MODEL: splunk
+LOGS_MODEL_CONFIG:
+  	producer: splunk
+  	splunk_config:
+    	host: http://<user_name>.remote.csb <1>
+    	port: 8089 <2>
+    	bearer_token: <bearer_token> <3>
+		url_scheme: <http/https> <4>
+    	verify_ssl: False <5>
+    	index_prefix: <splunk_log_index_name> <6>
```
Comment (harishsurf) [inline modules/proc_splunk-config.adoc:28]: Could you also include the newly added parameter `ssl_ca_path` which takes in a - "Relative container path to a single .pem file containing a CA"

File: modules/config-fields-actionlog.adoc [hunk 2]
```diff
@@ -61,10 +61,14 @@
...
-
-
-
+//splunk
+** **producer** [string]: `splunk`
+** **splunk_config** [object]: Logs model configuration for Splunk action logs or the Splunk cluster configuration
+*** **host** [string]: Splunk cluster endpoint.
+*** **port** [integer]: Splunk management cluster endpoint port.
+*** **bearer_token** [string]: The bearer token for Splunk.
+*** **use_ssl** [boolean]: Enable (`True`) or disable (`False`) TLS/SSL verification for HTTPS connections.
```
Comment (harishsurf) [inline modules/config-fields-actionlog.adoc:70]: `use_ssl` has been renamed to `verify_ssl`

File: modules/proc_splunk-config.adoc [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        ssl_ca_path: <location_to_ssl-ca-cert.pem> <7>
+---
+----
+<1> String. The Splunk cluster endpoint.
+<2> Integer. The Splunk management cluster endpoint port. Differs from the Splunk GUI hosted port. Can be found on the Splunk UI under *Settings* -> *Server Settings* -> *General Settings*.
+<3> String. The generated bearer token for Splunk.
+<4> String. The URL scheme for access the Splunk service. If Splunk is configured to use TLS/SSL, this must be `https`.
+<5> Boolean. Whether to enable TLS/SSL. Defaults to `true`.
+<6> String. The Splunk index prefix. Can be a new, or used, index. Can be created from the Splunk UI.
+<7> String. The relative container path to a single `.pem` file containing a certificate authority (CA) for TLS/SSL validation. Must be a full chain certificate, for example, a complete SSL/TLS certificate chain that includes the server certificate, intermediate certificate(s), and root certificate.
```
Comment (LiZhang19817) [inline modules/proc_splunk-config.adoc:38]: Based on the testing results, when only provide the rootCA Cert, it works well.

Comment (stevsmit) [reply modules/proc_splunk-config.adoc]: @LiZhang19817 fixed.

**Pull request merged**